frontend/Makefile.help

```diff
@@ -62,7 +62,9 @@ $(LIBCOMPILER_BUILD_DIR):
 	    $(LIB_CMAKE_ARG) \
 	    $(SDKROOT_ARG) \
 	    -DDYNO_LINTERS_CLANG_CMAKE_PATH='$(CLANG_CMAKE_DIR)' \
-	    -DBUILD_SHARED_LIBS=OFF
+	    -DBUILD_SHARED_LIBS=OFF \
```

Comment:
The indentation here looks kinda weird, should it not be aligned with the rest of the column?

Reply:
Weird, it's only unaligned in one of the diff modes...